### DISCUSSION TITLE
数式表示の崩れについて

### DISCUSSION THREAD

**Discussion by tanaka-asami415** [increments/qiita-discussions#417] [アナウンス]
2023年1月現在、数式表示が崩れるご報告を数件いただいております。
表示が崩れる際は、以下の方法を一度お試しいただけますと幸いです🙇

数式部分で右クリック > Math Settings > Math Renderer > HTML-CSSにチェックが入っている部分を「Common HTML」に変更

![suusiki](https://user-images.githubusercontent.com/103916185/213659930-5f69f80e-b558-47bb-ad38-0a753ec79c17.png)

**Comment by tsubolabo** [comment]
@tanaka-asami415 

　どのような環境で、どにような崩れがあるのかも教えていただけませんか？　Windows10、 Edge の環境下、数式がかなり多い自分の最新記事で確認してみましたが、特に問題は見つかりませんでした。

　それでも念のため、HTML-CSS から Common HTML に設定変更してみました。その結果は次のとおりです。

+ \dot, \ddot, \vec など、文字の上につく記号が、文字に接近し過ぎて見難くなります。
+ \text で数式中に挿入した漢字のフォントが大き目になり、英字とのバランスが崩れます。

個人的には、現状のままの HTML-CSS の方を好みます。

**Reply by tanaka-asami415** [reply]
@tsubolabo 
環境は色々あるようで特定には至っておりません🙇
崩れ方としては、数式用のフォントが上手くあたらず表示が変わってしまうようです。
こちらは崩れている方に向けての一時的な対処方法のご案内ですので、崩れていない場合は対応不要です！

**Reply by tsubolabo** [reply]
> 崩れていない場合は対応不要です！

分かっております。

　私の前コメントは婉曲過ぎて伝わり難かったようですが、「間違っても、邪魔なスクロールバーや見苦しいベースラインの不揃いが起こる以前の状態には戻さないで欲しい」という牽制です。宜しくお願いします。

**Reply by tanaka-asami415** [reply]
@tsubolabo 
> 「間違っても、邪魔なスクロールバーや見苦しいベースラインの不揃いが起こる以前の状態には戻さないで欲しい」という牽制です。宜しくお願いします。

承知しました！詳細ありがとうございます🙏
